### ISSUE TITLE
Copy/paste feature for exported variable values in inspector

### ISSUE THREAD

**Issue by lanalancia** [godotengine/godot-proposals#5872]
### Describe the project you are working on

A VR space game with a high mod support. 

### Describe the problem or limitation you are having in your project

My game have a lot of a framework tools which designed to help mod makers. Those tools have many exported variables like array with a per-shot spray pattern etc. Mod makers should be able to change their values without changing a script. Right now they should insert each vaue of array one per a time. For a spray pattern array, which might have 30 values, this isn't very comfortable.

Personally i need it for exported variables, but this feature might be useful for built-in variables as well (transform, visibility etc)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add a copy/paste functions in an existing context menu or make a new context menu for variable values. Right now "material" and "mesh" variables in inspector have it, but integer, float and boolean variables not. 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

My proposal is to make something like this 
![image](https://user-images.githubusercontent.com/96468948/204660724-4d536047-f3f7-44d5-a63d-248300d3b747.png)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

it could be worked around, but it makes unnecesary load. It reqires creating a child node with a unique script, something like

```
func _ready():
    get_parent().array_value = [1, 2, 3, 4, 5]
    queue_free()
```

### Is there a reason why this should be core and not an add-on in the asset library?

An addon requires a user to know about it availability. This is a quality of life feature. It doesn't removes or changes anything, it only adds new functionality. It doesn't displaces any UI elements. 

In other words, there's high probability users would like this feature and very low probability someone will be against it.

**Comment by KoBeWi** [comment]
But that's what Copy Property and Paste Property does, no?

**Comment by lanalancia** [comment]
Oh. It really what Copy Property is. That was kinda unclear. What about renaming it into "Copy Value" ?

**Comment by Mickeon** [comment]
> What about renaming it into "Copy Value" ?

https://github.com/godotengine/godot/pull/66689 does exactly that but the change has not been cherry-picked to 3.x just yet.

**Comment by KoBeWi** [comment]
Well it's resolved in 4.0, so closing.
